video/csputils.h

```diff
@@ -89,6 +93,7 @@ enum mp_csp_trc {
     MP_CSP_TRC_V_LOG,
     MP_CSP_TRC_S_LOG1,
     MP_CSP_TRC_S_LOG2,
+    MP_CSP_TRC_ST428,
```

Comment:
This one is added by https://github.com/mpv-player/mpv/pull/11200 so you don't have to duplicate work.